### ISSUE TITLE
Add Alt Text for all images uploaded to WP Content

### ISSUE THREAD

**Issue by NomeQ** [Trillium-Solutions/mendocino-wp-theme#20]
All images that have been uploaded to posts and pages should have alt text. These can be updated in wp-admin, in the media library.

**Comment by ekeaton** [comment]
updated on Production https://mendocinotransit.org/
